hardhat.config.js

```diff
@@ -0,0 +1,16 @@
+require('hardhat-circom');
+
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+module.exports = {
+  solidity: "0.6.7",
+  circom: {
+    ptau: "hash.ptau",
```

Comment:
Should we update to the hermez pot15 final ptau? (and keep their downloaded name)

Reply:
Yes